dm/syncer/syncer.go

```diff
@@ -1387,9 +1388,31 @@ func (s *Syncer) syncDDL(queueBucket string, db *dbconn.DBConn, ddlJobChan chan
 
 		if !ignore {
 			var affected int
+			var ddlCreateTime int64
+			ddlCreateTime = -1 // default when scan failed
+			row, err2 := db.QuerySQL(s.syncCtx, s.metricsProxies, "SELECT UNIX_TIMESTAMP()")
```

Comment:
Can we use QueryRowContext?

Reply:
Here `db` is `dbconn.DBConn`, it is not convenient to use `QueryRowContext`